snowpack/src/util.ts

```diff
@@ -261,6 +261,8 @@ export function findMatchingAliasEntry(
 ): {from: string; to: string; type: 'package' | 'path'} | undefined {
   // Only match bare module specifiers. relative and absolute imports should not match
   if (
+    spec === '.' ||
+    spec === '..' ||
```

Comment:
ha! is this supported in the browser? I'm not sure if I've ever used this myself, but I guess this is a valid path?

Reply:
There is a case that is using `import * from '.';` in utils/a.js to import some other function.